docs/reference/configuration/intents-operator/README.mdx

```diff
@@ -69,13 +69,16 @@ The intents operator can also be configured to process client intents *without*
 
 The Otterize credentials operator automatically registers client pods with the credentials service &mdash; either a SPIRE server, or the Otterize Cloud-managed credentials service &mdash; and writes the trusted credentials generated by that service into Kubernetes secrets for use by those pods. The intents operator takes `ClientIntents` with `type: kafka` and creates Kafka ACLs that grant the requested access to the cryptographic identities (SVIDs) created by the credentials operator.
 
-
 #### ACL creation and consumer groups
 A Kafka client may specify a consumer group ID when consuming a topic. When it does so, it requires DESCRIBE and READ access to the consumer group resource.
 To enable this, the intents operator creates an ACL enabling all consumers to read and describe all consumer groups.
 The permission check performed by the AclAuthorizer for a consumer group also takes into account whether the consumer has the appropriate access to the topic
 it is attempting to read, so the end result is that the topic ACLs determine actual access.
 
+### PostgreSQL users & access
+The intents operator automatically creates, updates, and deletes credentials in PostgreSQL databases according to the declared intents. It works together with the Otterize credentials operator to easily enable secure access to PostgreSQL from client pods, all in your Kubernetes cluster.
```

Comment:
I think we don't delete credentials from the PostgreSQL database when an intent is deleted.